cads_adaptors/tools/adaptor_tools.py

```diff
@@ -17,8 +17,8 @@ def get_adaptor_class(
             raise TypeError
         exec(setup_code)
         adaptor_class = eval(entry_point)
-    if not issubclass(adaptor_class, adaptor.AbstractAdaptor):
-        raise TypeError(f"{adaptor_class!r} is not subclass of AbstractAdaptor")
+    if not issubclass(adaptor_class, AbstractAdaptor):
+        raise TypeError(f"{adaptor_class!r} is not subclass of Base")
```

Comment:
This line needs to be reverted:
```suggestion
        raise TypeError(f"{adaptor_class!r} is not subclass of AbstractAdaptor")
```